lib/src/data/sports.dart

```diff
@@ -164,15 +163,19 @@ class SportsGame {
 	/// won, and which score to get depending on [isHome].
 	final Scores? scores;
 
+	/// The URL to the livestream for this game
```

Comment:
Also, whenever you have a nullable type, make sure to explain when it can be null and what should happen when it is.